docs/src/main/sphinx/security/file-system-access-control.rst

```diff
@@ -465,29 +504,195 @@ name, and allow ``alice`` and ``bob`` to use a group principal named as
+still be set to authorize this user to read or write to management information.
+The fixed management user only applies to HTTP by default. To enable the fixed
+user over HTTPS, set the ``management.user.https-enabled`` configuration
+property.
+
+.. _catalog-file-based-access-control:
+
+Catalog-level access control files
+==================================
+
+You can create JSON files for individual catalogs that define authorization
+rules specific to that catalog. To enable catalog-level access control files,
+add a connector-specific catalog configuration property that sets the
+authorization type to ``FILE`` and the ``security.config-file`` catalog
+configuration property that specifies the JSON rules file.
+
+For example, the following Iceberg catalog configuration properties use the
+``rules.json`` file for catalog-level access control:
+
+.. code-block:: properties
+
+  iceberg.security=FILE
+  security.config-file=etc/catalog/rules.json
+
+
+Catalog-level access control files are supported on a per-connector basis, refer
+to the connector documentation for more information.
+
+.. note::
+
+    These rules do not apply to system-defined tables in the
+    ``information_schema`` schema.
+
+Configure a catalog rules file
+------------------------------
+
+Catalog-level rules JSON files are composed of three sections, each of which is
+a list of rules that are matched in order from top to bottom. The user is
+granted the privileges from the first matching rule. All regexes default to
+``.*`` if not specified.
+
+Schema rules
+^^^^^^^^^^^^
+
+These rules govern who is considered an owner of a schema.
+
+* ``user`` (optional): regex to match against user name.
+* ``group`` (optional): regex to match against every user group the user belongs
+  to.
+* ``schema`` (optional): regex to match against schema name.
+* ``owner`` (required): boolean indicating ownership.
+
+Table rules
+^^^^^^^^^^^
+
+These rules govern the privileges granted on specific tables.
+
+* ``user`` (optional): regex to match against user name.
+* ``group`` (optional): regex to match against every user group the user belongs
+  to.
+* ``schema`` (optional): regex to match against schema name.
+* ``table`` (optional): regex to match against table name.
+* ``privileges`` (required): zero or more of ``SELECT``, ``INSERT``,
+  ``DELETE``, ``OWNERSHIP``, ``GRANT_SELECT``.
+* ``columns`` (optional): list of column constraints.
+* ``filter`` (optional): boolean filter expression for the table.
+* ``filter_environment`` (optional): environment use during filter evaluation.
+
+Column constraint
+~~~~~~~~~~~~~~~~~
+
+These constraints can be used to restrict access to column data.
+
+* ``name``: name of the column.
+* ``allow`` (optional): if false, column can not be accessed.
+* ``mask`` (optional): mask expression applied to column.
+* ``mask_environment`` (optional): environment use during mask evaluation.
+
+Filter and mask environment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
```

Comment:
Add some text to clarify this is for the value of `mask_environment` and `filter_environment`.